src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -578,69 +578,36 @@
-    text: |
-        az acr task create -t hello-world:{{.Run.ID}} -n hello-world -r MyRegistry \\
-            -c https://github.com/Azure-Samples/acr-tasks.git#:multipleRegistries -f testtask.yaml \\
-            --assign-identity
-  - name: Create a Linux multi-step task from a public GitHub repository with a git commit trigger and with a user-assigned managed identity
+            --commit-trigger-enabled false --pull-request-trigger-enabled false \\
+            --platform Windows/amd64
+  - name: Create a Linux multi-step task from a public GitHub repository with with both system-assigned and user-assigned managed identities and with a git commit, pull request, base image, and timer trigger that runs the task at noon on Mondays through Fridays with the trigger name provided.
     text: |
         az acr task create -t hello-world:{{.Run.ID}} -n hello-world -r MyRegistry \\
-            -c https://github.com/Azure-Samples/acr-tasks.git#:multipleRegistries -f testtask.yaml \\
-            --assign-identity "/subscriptions/<SUBSCRIPTON ID>/resourcegroups/myResourceGroup/providers/Microsoft.ManagedIdentity/userAssignedIdentities/myUserAssignedIdentitiy"
-  - name: Create a Linux multi-step task from a public GitHub repository with a git commit trigger and with both system-assigned and user-assigned managed identities
-    text: |
-        az acr task create -t hello-world:{{.Run.ID}} -n hello-world -r MyRegistry \\
-            -c https://github.com/Azure-Samples/acr-tasks.git#:multipleRegistries -f testtask.yaml \\
-            --assign-identity [system] "/subscriptions/<SUBSCRIPTON ID>/resourcegroups/myResourceGroup/providers/Microsoft.ManagedIdentity/userAssignedIdentities/myUserAssignedIdentitiy"
-  - name: Create a Linux multi-step task from a public GitHub repository with a system-assigned managed identity and a timer trigger that runs that task at noon on Mondays through Fridays using the default trigger name.
-    text: |
-        az acr task create -t hello-world:{{.Run.ID}} -n hello-world -r MyRegistry \\
-            -c https://github.com/Azure-Samples/acr-tasks.git#:multipleRegistries -f testtask.yaml \\
-            --commit-trigger-enabled false --schedule "0 12 * * Mon-Fri"\\
-            --assign-identity
-  - name: Create a Linux task from a public GitHub repository which builds the hello-world image with a git commit trigger and a timer trigger that runs that task at noon on Mondays through Fridays with the trigger name provided.
-    text: |
-        az acr task create -t hello-world:{{.Run.ID}} -n hello-world -r MyRegistry \\
-            -c https://github.com/Azure-Samples/acr-build-helloworld-node.git -f Dockerfile \\
+            -c https://github.com/Azure-Samples/acr-tasks.git#:multipleRegistries \\
+            -f testtask.yaml --assign-identity [system] "<myIdentityResourceId>" \\
```

Comment:
@rosanch you will need to explicitly enable the pr trigger here as it is disabled by default.